### PR TITLE
fix: [PL 20548]: Center align Placeholder of tags component

### PR DESCRIPTION
**Pull request by mayankVermaHarness**
**Summary**
Center align Placeholder of tags component

**Jira Links:**
https://harness.atlassian.net/browse/PL-20548

**Screenshots:**

**Before:**
image

**After:**
image

**Before:**
image

**After:**
image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Review by abhinavrastogi-harness — APPROVED** [review]
have you tested this with large number of tags, when it renders in multiple lines?

File: src/components/FormikForm/FormikForm.css [hunk 1]
```diff
@@ -80,7 +80,7 @@
       }
 
       .bp3-tag-input-values {
-        margin: 1px 7px 2px 1px !important;
+        margin: 2px 7px 0 1px !important;
```
Comment (avinashmadhwani02) [inline src/components/FormikForm/FormikForm.css:83]: Possible to center align without margin? 

We will have to change these values if in any case height of the input changes

**Review by avinashmadhwani02 — APPROVED** [review]

**Comment by mayankVermaHarness** [comment]
@abhinavrastogi-harness I did check it, there's a max-height of 70px after which it overflows.
image


The placeholder stops being visible even if a single tag is present.
image

File: src/components/FormikForm/FormikForm.css (hunk 1 above)
Comment (mayankVermaHarness) [reply src/components/FormikForm/FormikForm.css:83]: Not possible right now.

Also, blueprintjs internally has a min-height of 30px for tag input. If ever needed, we'll have to overwrite blueprint's styles too.

**Pull request merged**